Drop-In/src/main/java/com/braintreepayments/api/dropin/utils/PaymentMethodType.java

```diff
@@ -25,7 +25,7 @@
     UNIONPAY(CardType.UNIONPAY.getFrontResource(), R.drawable.bt_ic_vaulted_unionpay, R.string.bt_descriptor_unionpay, "UnionPay", CardType.UNIONPAY),
     HIPER(CardType.HIPER.getFrontResource(), R.drawable.bt_ic_vaulted_hiper, R.string.bt_descriptor_hiper, "Hiper", CardType.HIPER),
     HIPERCARD(CardType.HIPERCARD.getFrontResource(), R.drawable.bt_ic_vaulted_hipercard, R.string.bt_descriptor_hipercard, "Hipercard", CardType.HIPERCARD),
-    UNKNOWN(CardType.UNKNOWN.getFrontResource(), R.drawable.bt_ic_vaulted_unknown, R.string.bt_descriptor_unknown, "Unknown", CardType.UNKNOWN);
+    UNKNOWN(R.drawable.bt_ic_vaulted_unknown, R.drawable.bt_ic_vaulted_unknown, R.string.bt_descriptor_unknown, "Unknown", CardType.UNKNOWN);
```

Comment:
Can we remove the naming of `vaulted` to describe this drawable? Since it is used in the actual payment selection view controller for drop-in and not just the vaulted card list.

Reply:
Though unlikely, it might be possible that folks rely on that naming/file for custom UI?

Reply:
Hm, I didn't think that was a use case. 

I will leave it up to your discretion, though would advocate for renaming if possible.

Reply:
maybe we make a major version note to rename?

Reply:
Yeah I'm down. Would this go on the major version bump wish list in the knowledge repo?